frontend/packages/dev-console/src/components/topology/operators/TopologyOperatorBackedResources.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import OperatorBackedOwnerReferences from '@console/internal/components/utils';
+import { OverviewItem } from '@console/shared';
+import { PodsOverview } from '@console/internal/components/overview/pods-overview';
+import { BuildOverview } from '@console/internal/components/overview/build-overview';
+import { NetworkingOverview } from '@console/internal/components/overview/networking-overview';
+import { TopologyDataObject } from '../topology-types';
+
+type TopologyOperatorBackedResourcesProps = {
+  item: TopologyDataObject;
+};
+
+const TopologyOperatorBackedResources: React.FC<TopologyOperatorBackedResourcesProps> = ({
+  item,
+}) => {
+  const combinedResources = item.groupResources.reduce<OverviewItem>(
```

Comment:
will `groupResources` be always defined as `groupResources` is optional param in `TopologyDataObject` ? so may be good to have a check

Reply:
Added default `[]` value during destructure.